pubspec.yaml

```diff
@@ -43,6 +43,7 @@ dependencies:
   url_launcher: ^6.0.0
 
   accessibility_tools: ^1.0.0
+  collection: ^1.0.0
```

Comment:
nits: Please add to line 36.

Reply:
Done :)
[style: change line](https://github.com/FlutterKaigi/conference-app-2023/pull/142/commits/6cefe9aa98b5829355a1ca11fae396f938a5627d)